post_office/template/backends/post_office.py

```diff
@@ -20,7 +20,7 @@ def attach_related(self, email_message):
             email_message.attach(attachment)
 
 
```

Comment:
Can we rename this file from `post_office/template/backends/post_office.py` to `post_office/template/backends.py`?